animalai/arenas/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os
```

Comment:
Cause they are the simplest possible arena's. Just to test everything works. We should maybe prefix with `example` to make this more clear.

Reply:
I'm not sure why we have these two arenas to test things as they're too basic and do not really test anything but loading the arena with the agent and a single reward, with fixed and random positions.

Reply:
Well, that tests a lot already right? As a user, this would be a very useful signal for me, in that I can successfully load non-empty arenas (and start playing / training agents). These very tasks are actually the tasks you would use to sanity check your agent implementation: Can it chase a simple green goal?

See also https://github.com/Kinds-of-Intelligence-CFI/animal-ai-python/pull/6#discussion_r1469731247

Reply:
The logic for starting up AAI already contains code to handle instances where there are no arenas defined or the arenas are empty (upcoming in the next version of animalai). I think perhaps we could implement tests specifically with starting training mode, and with the example of an agent chasing a reward, this would be a better test to carry out.

Thoughts on this?

Reply:
I think it's good to distinguish between testing (for us to ensure AAI works) and sanity checking (for new users / setups to check things are working). I think these can be useful for both.
- If we want people to use these for sanity checks we should document that (e.g. the last step in the setup instructions is to run one of these)
- For testing I think a 'train to good reward on a really basic env' E2E test sounds good. An alternative that would mean we don't have to do training (since we don't need to test that) would be to use fixed action sequences. If we wanted to set a very high bar we could use these for screenshot testing (load into the env and the visual output has to look exactly like this)
I think someone mentioned renaming to example-arenas or something like that. I think that's a good idea since it makes it clear at a glance there's no functionality here

Reply:
I see what you mean. I misunderstood it. Yep totally agree